src/intl/compiled/en.json

```diff
@@ -479,6 +485,16 @@
       "value": "Install deposit-cli tool"
     }
   ],
+  "3Bo3k6": [
+    {
+      "type": 1,
+      "value": "withdrawalCredentials"
+    },
+    {
+      "type": 0,
+      "value": " field is the 32-byte value in the deposit, for verifying the destination of valid withdrawals. Currently, there are two types of withdrawals: BLS withdrawal and Eth1 address withdrawal."
```

Comment:
Grammatically something is off to me here:

"Withdrawal Credentials field is the 32-byte value in the deposit, for verifying the destination of valid withdrawals."

What about something like:
* "Withdrawal Credentials is a 32-byte field in the deposit"
* "The Withdrawal Credentials field is the 32-byte value in the deposit"